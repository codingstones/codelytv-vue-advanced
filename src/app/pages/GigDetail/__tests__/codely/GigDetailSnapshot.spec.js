import GigDetail from '@/app/pages/GigDetail/GigDetail.vue'
import { cloneProductionStore, Wrap } from '../../../../../../test/helpers'
import PageObject from '../../../../__page_objects__/PageObject'

jest.mock('@/app/services/jota-api')

describe('Gig Detail', () => {

  it('matches snapshot', async () => {
    let store = cloneProductionStore()
    const wrapper = Wrap(GigDetail)
      .withStore(store)
      .mount()
    const page = new PageObject(wrapper)
    await page.wait()

    page.matchSnapshot()
  })
})
