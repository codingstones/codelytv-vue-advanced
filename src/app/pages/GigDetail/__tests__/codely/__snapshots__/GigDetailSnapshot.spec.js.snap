// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gig Detail matches snapshot 1`] = `
<div class="full-width" style="">
    <div class="q-card">
        <div class="q-card-primary q-card-container row no-wrap">
            <div class="col column">
                <div class="q-card-title">
                    <h3>Anarchy in the Jota S</h3>
                </div>
                <div class="q-card-subtitle"></div>
            </div>
            <div class="col-auto self-center q-card-title-extra"></div>
        </div>
        <div class="q-card-main q-card-container">
            <!---->
            <div class="block"><strong>Lunes, 18 De Septiembre</strong> a las <strong>19:30 h.</strong></div> <button class="q-btn row inline flex-center q-focusable q-hoverable relative-position q-btn-rectangle q-btn-standard bg-secondary text-white"><div class="desktop-only q-focus-helper"></div> <!----> <span class="q-btn-inner row col flex-center"><!----> <i aria-hidden="true" class="q-icon material-icons">event</i>  Add it to your calendar  <!----></span></button>
            <div class="block">
                <h4>SCBCN</h4> <span>BCN</span></div> <button class="q-btn row inline flex-center q-focusable q-hoverable relative-position q-btn-rectangle q-btn-standard bg-secondary text-white"><div class="desktop-only q-focus-helper"></div> <!----> <span class="q-btn-inner row col flex-center"><!----> <i aria-hidden="true" class="q-icon material-icons">directions</i> How to get there?  <!----></span></button>            <br><br>
            <div class="block">
                Coding Stones cantando Joticas
            </div> <button class="q-btn row inline flex-center q-focusable q-hoverable relative-position q-btn-rectangle q-btn-standard bg-secondary text-white"><div class="desktop-only q-focus-helper"></div> <!----> <span class="q-btn-inner row col flex-center"><!----> <i aria-hidden="true" class="q-icon material-icons">share</i> Share it with your friends  <!----></span></button></div>
        <div class="q-card-media relative-position"><img src="http://www.zaragoza.es/cont/paginas/actividades/imagen/web_320x480px.png"></div>
    </div>
</div>
`;
